xtext-website/documentation/303_runtime_concepts.md

```diff
@@ -103,9 +103,9 @@ To customize error messages please see [IConcreteSyntaxDiagnosticProvider]({{sit
 
 ### Custom Validation {#custom-validation}
 
-In addition to the afore mentioned kinds of validation, which are more or less done automatically, you can specify additional constraints specific for your Ecore model. The Xtext language generator will provide you with two Java classes. The first is an abstract class generated to *src-gen/* which extends the library class [AbstractDeclarativeValidator]({{site.src.xtext_core}}/org.eclipse.xtext/src/org/eclipse/xtext/validation/AbstractDeclarativeValidator.java). This one just registers the EPackages for which this validator introduces constraints. The other class is a subclass of that abstract class and is generated to the *src/* folder in order to be edited by you. That is where you put the constraints in.
+In addition to the aformentioned kinds of validation, which are more or less done automatically, you can specify additional constraints specific to your Ecore model. The Xtext language generator will provide you with two Java classes. The first is an abstract class generated in *src-gen/* which extends the library class [AbstractDeclarativeValidator]({{site.src.xtext_core}}/org.eclipse.xtext/src/org/eclipse/xtext/validation/AbstractDeclarativeValidator.java). This class  registers the EPackages for which this validator introduces constraints. The other class is a subclass of that abstract class and is generated in the *src/* folder to be edited by you. That is where you put the constraints in.
```

Comment:
I think "this one" is clear here, but "this class" is fine also. There is a double space followed after "class", thoug.